samples/ck-rx65n/basic-sample/include/app_config.h

```diff
@@ -1,11 +1,24 @@
 //
 // Copyright: Avnet 2022
-// Created by Russell Bateman <rbateman@witekio.com> on 11/18/22.
 //
 
 #ifndef APP_CONFIG_H
 #define APP_CONFIG_H
 
+#define IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY 1
```

Comment:
Why is this variable called "interactive symmetric key"? It seems to be not just a about symmetric key.

Reply:
I've renamed it INPUT_SYMMETRIC_KEY -- and added comments, hopefully that should make the intention clearer.

Reply:
What I meant to say is that code surrounded by this flag in the part where you are setting it includes both Symmetric Key and X509 auth, so calling it *anything*key*anything* seems odd